lib/json5/read.ml

```diff
@@ -0,0 +1,34 @@
+let ( let* ) = Result.bind
```

Comment:
It could make sense to make a module `lib/json5/let_syntax.ml` (or so) with a

```ocaml
module Result = struct
  let (let * ) = Result.bind
end
```

Then it can be `open`-ed wherever this is useful.